components/Footer.js

```diff
@@ -1,56 +1,131 @@
+            >
+              <path d="M0 0h24v24H0z" fill="none" />
+              <path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm5 11h-4v4h-2v-4H7v-2h4V7h2v4h4v2z" />
+            </svg>
+          ) : (
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              height="24px"
+              viewBox="0 0 24 24"
+              width="24px"
+              fill="white"
+            >
+              <path d="M0 0h24v24H0z" fill="none" />
+              <path d="M13 7h-2v4H7v2h4v4h2v-4h4v-2h-4V7zm-1-5C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8z" />
+            </svg>
+          )}
+          <NavIconText>Teilen</NavIconText>
+        </StyledNavItem>
+      </NavLink>
+      <NavLink href={`/favorites`}>
+        <StyledNavItem>
+          {pathname === "/favorites" ? (
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              height="24px"
+              viewBox="0 0 24 24"
+              width="24px"
+              fill="white"
+            >
+              <path d="M0 0h24v24H0z" fill="none" />
+              <path d="M12 21.35l-1.45-1.32C5.4 15.36 2 12.28 2 8.5 2 5.42 4.42 3 7.5 3c1.74 0 3.41.81 4.5 2.09C13.09 3.81 14.76 3 16.5 3 19.58 3 22 5.42 22 8.5c0 3.78-3.4 6.86-8.55 11.54L12 21.35z" />
+            </svg>
+          ) : (
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              height="24px"
+              viewBox="0 0 24 24"
+              width="24px"
+              fill="white"
+            >
+              <path d="M0 0h24v24H0z" fill="none" />
+              <path d="M16.5 3c-1.74 0-3.41.81-4.5 2.09C10.91 3.81 9.24 3 7.5 3 4.42 3 2 5.42 2 8.5c0 3.78 3.4 6.86 8.55 11.54L12 21.35l1.45-1.32C18.6 15.36 22 12.28 22 8.5 22 5.42 19.58 3 16.5 3zm-4.4 15.55l-.1.1-.1-.1C7.14 14.24 4 11.39 4 8.5 4 6.5 5.5 5 7.5 5c1.54 0 3.04.99 3.57 2.36h1.87C13.46 5.99 14.96 5 16.5 5c2 0 3.5 1.5 3.5 3.5 0 2.89-3.14 5.74-7.9 10.05z" />
+            </svg>
+          )}
+          <NavIconText>Favoriten</NavIconText>
+        </StyledNavItem>
+      </NavLink>
     </StyledFooter>
   );
 }
 
+/* ↓↓ Styling ↓↓ */
+
 const StyledFooter = styled.nav`
-  background-color: #4d5b5b;
-  padding: 8px;
+  background-image: linear-gradient(to right, #2f3737 0%, #4d5b5b 100%);
+  padding: 2px 20px 1px 30px;
   position: fixed;
   bottom: 0;
   width: 100%;
   display: flex;
-  justify-content: space-around;
+  justify-content: space-between;
   box-shadow: rgba(99, 99, 99, 0.2) 0px 2px 8px 0px;
+  z-index: 100;
+`;
+
+const StyledNavItem = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 3px;
+
+  :hover {
+    filter: drop-shadow(3px 3px 3px rgb(0 0 0 / 0.9));
+  }
+`;
+
+const NavLink = styled(Link)`
+  text-decoration: none;
+  filter: drop-shadow(3px 3px 3px rgb(0 0 0 / 0.2));
+`;
+
+const NavIconText = styled.p`
+  color: white;
```

Comment:
Same as above, it is better to use a hex code instead of a named color :) 
```suggestion
  color: #ffffff;
```